src/JoyOI.VirtualJudge.LeetCode/LeetcodePullProblemBodyActor.cs

```diff
@@ -81,6 +81,7 @@ public static void Main()
                 Source = "LeetCode",
                 OriginUrl = baseUrl + problemUri,
                 Templates = templates,
+                TemplatesError = File.ReadAllText("stderr.txt"),
```

Comment:
Please detect the runner exit code, if exit code is non-zero, please throw the exception, please don't return the body.

Reply:
Also, we don't have the field which named TemplatesError and Templates, please take a look into ProblemJson class in OnlineJudge repo.